enhancements/installer/aws-outpost-remote-workers.md

```diff
@@ -0,0 +1,209 @@
+  - "@pkliczewski"
+reviewers:
+  - "@sdodson"
+approvers:
+  - "@sdodson"
+api-approvers:
+  - None
+creation-date: 2022-10-11
+last-updated: 2022-10-11
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-9617
+  - https://issues.redhat.com/browse/ECOPROJECT-866
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+
+# Remote workers on AWS outpost
+
+## Summary
+
+Running workload closer to end users or leveraging data locality seems to be
+important for many customers. That is why AWS came up with fully [managed
+service](https://docs.aws.amazon.com/outposts/latest/userguide/how-outposts-works.html)
+that extends AWS infrastructure and run on customer's premises. This enhancement
+proposes extending how we deploy OCP on AWS platform so it ispossible to run on
+AWS Outpost.
+
+The work is divided into several phases:
+- Phase 0: Research. Understand what is needed to deploy OCP on AWS Outpost.
+  No code changes needed and we only provide documentation on how to customize
+  generated manifests and list known limitations.
+- Phase 1: Improve cluster deployment on AWS Ouptost by implementing necessary
+  changes and provide mitigations to know limitations.
+- Phase 2: Add OCP on AWS Outpost support to ROSA for regular clusters and HyperShift.
+
+## Motivation
+
+Users might want to use OCP on AWS Outpost running in their own data center.
+AWS Outposts enables customers to build and run applications on premises using
+the same programming interfaces as in AWS Regions, while using local compute
+and storage resources for lower latency and local data processing needs.
+
+### User Stories
+
+Phase 0:
+- As a cluster admin I want to follow OCP documentation to deploy OCP on AWS Outpost
+  where my master nodes are running in the regular AWS region and workers in AWS Outpost.
+
+Phase 1:
+- As a cluster admin I want to use an OCP installer to deploy OCP on AWS Outpost.
+  (Cluster deployment topology depends on mitigating lack of support for NLB)
+
+Phase 2:
+- As a cluster admin I want to use ROSA cli or UI to deploy OCP/Hypershift on AWS Outpost.
+  (OCP deployment topology similar as in Phase 1)
+
+
+### Goals
+
+Phase 0:
+- Admin can follow documentation to customize installer generated manifests to deploy
+  OCP on AWS Outpost
+- As post installation step admin can configure ALB on AWS Outpost as workload ingress
+- Admin can check list of limitations or not supported features
+
+Phase 1:
+- Admin can use OCP installer to deploy OCP on AWS Outpost
+- After installation only gp2 storageclass is available and ingress is configured
+
+Phase 2:
+- Admin can use ROSA cli or UI to deploy OCP/Hypershift on AWS Outpost
+
+### Non-Goals
+
+We do not intend to implement AWS Outpost specific features in OCP as well us we
+do not plan to handle long lasting networking issues between regular AWS region andan outpost.
+
+## Proposal
+
+Phase 0:
+We want to provide documented manual procedure on how to deploy an OCP cluster in a regular
```

Comment:
I see that we've got a docs PR open for the procedure, I don't want to ask you to copy the entire contents of the procedure here but I also don't think it's possible to review this enhancement with an eye toward ensuring that we're creating a supportable infrastructure without relevant teams being able to follow exactly what's being done.  Can we link to the proposed docs PR and the demo so that folks can better understand the impelementation details?

Reply:
sure, I will add a link